package.json

```diff
@@ -0,0 +1,19 @@
+{
+  "name": "dredd-docker",
+  "version": "1.0.0",
+  "description": "Available at [Docker Hub](https://hub.docker.com/r/apiaryio/dredd/). For further information on Dredd, see [the documentation](http://dredd.org/en/latest/).",
+  "main": "index.js",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/apiaryio/dredd-docker.git"
+  },
+  "author": "",
+  "license": "MIT",
```

Comment:
MIT license, but no license file in this changeset.

Reply:
already in master